packages/icons/scripts/icons.js

```diff
@@ -0,0 +1,35 @@
+const fs = require("fs");
+const path = require("path");
+
+const { pascalCase } = require("case-anything");
+const dotenv = require("dotenv");
+const { fetchImages } = require("figma-tools");
+const prettier = require("prettier");
+const svgtojsx = require("svg-to-jsx");
+
+dotenv.config({ path: "../../.env" });
+
+const pretterOptions = JSON.parse(fs.readFileSync("../../.prettierrc", "utf-8"));
+
+console.log("Fetching icons...");
+
+fetchImages({
+  fileId: "ASas6u2DMihEEzw8jPT1XC",
+  format: "svg",
+  filter: component => component.pageName === "0.3 Icons",
+}).then(async svgs => {
+  console.log("Building icons...");
+
+  const jsx = await Promise.all(svgs.map(svg => svgtojsx(svg.buffer)));
+  const data = svgs
+    .map((svg, index) => `export const ${pascalCase(svg.name)}Icon = () => ${jsx[index]}\n`)
```

Comment:
Worth adding a warning if we ever encounter a duplicate key? I imagine that would be unexpected but 🤷🏼